### PR TITLE
Add UFZ as partner

### LINKED ISSUE #56: Add UFZ as Partner
> The Helmholtz Centre for Environmental Research – UFZ is one of the world’s leading research centres in the field of environmental research, enjoying high social recognition. Biodiversity, functioning ecosystems, clean water and intact soils all make up our natural resource base. In the face of global change, employees at the UFZ are united by the goal of demonstrating ways to combine societal development with a healthy environment. As a reliable partner, the UFZ supports the political arena, the economy and the general public to better understand the consequences of human actions on the environment and to develop options for social decision - making processes.

### PR DESCRIPTION
**Pull request by FWirtz**
Refactor sponsor-kilo to sponsor-mega

Fixes #56

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -229,11 +229,14 @@ <h1 class="text-center">Partners</h1>
 				</div>
 			</div>
 
-			<div class="row sponsors-kilo">
+			<div class="row sponsors-mega">
 				<div class="col-md-12 top50">
-					<a href="http://www.pik-potsdam.de/copan/" class="sponsor" target="_blank">
+					<a href="https://www.pik-potsdam.de/copan/" class="sponsor" target="_blank">
```
Comment (jak-ing) [inline index.html]: 👍

File: partner-information.html [hunk 2]
```diff
@@ -99,6 +99,17 @@ <h3>Potsdam Institute for Climate Impact Research</h3>
 				</div>
 			</div>
 
+			<div class="row sponsor-information">
+				<div class="col-md-4">
+					<img src="../hackhpi_assets/2018-partners/UFZ_white.jpg" class="img-responsive">
+				</div>
+
+				<div class="col-md-8">
+				<h3>Helmholtz Centre for Environmental Research – UFZ</h3>
```
Comment (jak-ing) [inline partner-information.html]: On their website it's `Helmholtz-Centre for Environmental Research - UfZ`, which version should we use?

Comment (FWirtz) [reply partner-information.html]: huh good question, would leave it as is as that is extracted from their mission statement

Comment (jak-ing) [reply partner-information.html]: Alright.

**Pull request merged**